docs/guide/walkthrough/Project/Curation.md

```diff
@@ -0,0 +1,171 @@
+::: 
+
+### PRISMA Workflow
+
+PRISMA stands for the **Preferred Reporting Items for Systematic Review and Meta-Analyses**. The
+[PRISMA guidelines](http://www.prisma-statement.org/?AspxAutoDetectCookieSupport=1) are a set of rules for reporting a 
+systematic review, and are the gold standard for producing a proper, rigorous, and transparent meta-analysis.
+
+If you are trying to create a rigorous [manual meta-analysis](/compose-docs/tutorial/manual), or if you are trying to create a
+[PRISMA compliant semi-automated meta-analysis](/compose-docs/tutorial/semi-automated), you will want to select this option.
+
+When you select this option, neurosynth-compose will automatically initialize the curation step with 4 distinct columns as
+dictated by PRISMA guidelines: **Identification, Screening, Eligibility, Included**.
+
+After importing studies into neurosynth, they will be placed into the identification column and studies are triaged from there.
+
+:::info PRISMA Summary
+The **Identification** column is where all imported studies are deposited into initially. As alluded to by the name, the identification
+column is where you identify all records yielded from your search. In this column, you wil exclude studies purely based on whether they are 
+duplicates of existing studies. All other studies are promoted.
+
+The **Screening** column consists of all records that have been screened for duplicates. In this column, you want to review the titles/abstracts of studies 
+and exclude purely based on whether they are irrelevant to your research question or domain. All other studies are promoted. 
+
+The **Eligibility** column consists of all records that have been screened for duplicates and irrelevant content. In this column, you want to 
+review the full text of studies and exclude based on whether the study described aligns with the standards of the meta-analysis itself. Reasons 
+for exclusion may include wrong setting, wrong patient population, wrong intervention, wrong paragdigm, etc.
+
+The **Included** column consists of all records that have passed previous levels of exclusion and can be considered for the next step of the project.
+
+For more information, consult the [PRISMA guidelines](http://www.prisma-statement.org/?AspxAutoDetectCookieSupport=1).
+:::
+
+### Simple Workflow
+
+If you want to create a [semi-automated meta-analysis](/compose-docs/tutorial/semi-automated) (i.e. perform manual review on a large-scale study search), you'll want to select this option.
+
+This workflow is initalized with only two columns. As before, the left most column is where studies will be placed when they are imported. However, unlike a full PRISMA workflow, all exclusion occurs in this column. All studies not excluded are then promoted to the right most column for inclusion into the meta-analysis.
+
+### Custom Workflow
```

Comment:
I removed the ability to add arbitrary columns in the UI because im convinced that there is no use case for it.
For now, we should focus on making our current offering (PRISMA compliant meta-analysis OR simple) rock solid. If users later tell us that they want this, we can add it back.